test/unit/__snapshots__/subscriptions.test.ts.snap

```diff
@@ -28,7 +28,8 @@ Array [
 
 exports[`sellers deregisterDestination returns the standard response if deregistration is succesful 1`] = `
 Array [
-  "",
+  "
```

Comment:
Why do you think that is? Is it that Amazon returning a new line?

Reply:
I checked and it's due to the difference of how this
```xml
   <DeregisterDestinationResult>
    </DeregisterDestinationResult>
```
and this
```xml
<DeregisterDestinationResult></DeregisterDestinationResult>
```
or this
```xml
<DeregisterDestinationResult/>
```
gets parsed

Reply:
The first one gets parsed to `"\n    "` and the other two get parsed to `""`